packages/manager/modules/pci/src/projects/project/storages/databases/add/add.utils.js

```diff
@@ -14,6 +16,21 @@ export function getOrderDataFromModel(model) {
         : model.disk.initialSize,
     },
   };
+
+  if (model.restoreMode) {
+    if (model.restoreMode === RESTORE_MODES.BACKUP) {
+      orderData.forkFrom = {
+        serviceId: model.databaseId,
+        backupId: model.backupId,
+      };
+    } else {
+      orderData.forkFrom = {
+        serviceId: model.databaseId,
+        pointInTime: model.timestamp,
+      };
+    }
+  }
```

Comment:
```suggestion
  if (model.restoreMode) {
      orderData.forkFrom = {
        serviceId: model.databaseId,
        ...(model.restoreMode === RESTORE_MODES.BACKUP ? {backupId: model.backupId} : {pointInTime: model.timestamp})
      };
  }
```

Reply:
Hi, I forget a case here, so I went for a switch instead. Sorry.